packages/warriorjs-core/src/Unit.js

```diff
@@ -92,12 +92,13 @@ class Unit {
    */
   performTurn() {
     if (this.isAlive()) {
-      this.effects.forEach(effect => { 
-        if(this.isAlive()) {
-          effect.passTurn(); 
+      for (let effect of this.effects) {
+        effect.passTurn();
+        if (this.isAlive()) {
```

Comment:
hello @olistic , i have a doubt about this. Is the isAlive check needed after passTurn()? and
you suggested to use return to stop looping. how do i do that? with return -1? I'm a bit confused.
thank you.